django/db/migrations/autodetector.py

```diff
@@ -95,7 +95,7 @@ def only_relation_agnostic_fields(self, fields):
         fields_def = []
         for name, field in sorted(fields.items()):
             deconstruction = self.deep_deconstruct(field)
-            if field.remote_field and field.remote_field.model:
+            if field.remote_field and field.remote_field.model and 'to' in deconstruction[2]:
                 del deconstruction[2]['to']
```

Comment:
I guess we could `.pop('to', None)` instead to avoid the dual lookup?

```suggestion
            if field.remote_field and field.remote_field.model:
                deconstruction[2].pop('to', None)
```

Reply:
That was my original patch but I thought it seemed weird to have that meaningless second argument to `pop`.
But I hadn't thought about the performance aspect and in this case it makes sense so I've changed it.